app/views/companies/index.html.slim

```diff
@@ -0,0 +1,11 @@
+.container
+  .section
+    - @companies.group_by(&:supporter?).reverse_each do |supporter, companies|
```

Comment:
Why not sort companies already in `Company.all` method? Then you don't need any logic here, just iterate